### DISCUSSION TITLE
What is [initialize] vector='model' and what are the differences between stock models?

### DISCUSSION THREAD

**Discussion by source19069** [explosion/spaCy#8336] [Language Support]
We have a trained a bioinformatics model using spacy for inference of numeric test values associated with specific disease treatments, with good scores. The input datasets were created using the spacy _en_core_web_sm_ model. 

We'd now like to compare our results by generating other models using different stock models as a base, so we'd like to understand the difference between, say the _en_core_web_sm_ and _en_core_web_lg_ models.

Obviously we can just rebuild the datasets using different as bases and train new models, but what is the effect of using 

[initialize]
vector='model' 

in the config file during training?

We used 'en_core_web_lg' for the existing model training and the statistics and results look good, but we would really like to know what effect this parameter actually has, if any,  on the trained model.

Thank you.

**Comment by polm** [comment]
As noted [in the docs](https://spacy.io/api/data-formats#:~:text=name%20or%20path%20of%20pipeline%20containing%20pretrained%20word%20vectors%20to%20use), that just determines which static vectors are included in your model. Depending on your settings those can be used as input features during training. See the [static vectors docs](https://spacy.io/usage/embeddings-transformers#static-vectors).

The difference between the small, medium, and large models is whether they include word vectors, how many they include, and the size of the model. See [here](https://spacy.io/models/en) for details.
